### DISCUSSION TITLE
Scrum Prep Milestone #2 - Third Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#274] [Weekly Retrospective]
Weekly Development Update: First week of Milestone 2's second sprint  (Or third week of the second milestone)

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, October 11, 2024 @ 11am.

Thank you for your collaboration!

As for my collaboration:

I updated the lecture topic document to help you all with your lecture topics. I also helped close and grade issues for the first sprint. We held a weekly meeting as well on Tuesday with @evnd04 and the team leaders.

**Comment by noel-vargas** [comment]
Created a diagram to show how the profile management tab will work under the hood. Basically showing how the components and how the data flows

**Comment by JorgeC08** [comment]
Finished a documentation task, moving over to find why I can't connect to docker when using Expo in the incorrecto password handling issue.

**Comment by CaribbeanCool** [comment]
I started working on 2 of my tasks and working on the Lecture Topic Task

**Comment by evnd04** [comment]
This week ive been a little inactive due to various exams and projects. I did work on approving tasks though and left feedback on many.

**Comment by javier-rivera8** [comment]
I designed the report deletion page by adding a scroll view to make it easier for users to delete reports. I also started working on its implementation.

**Comment by Alejandro-DeJesus** [comment]
This week, I submitted a pull request for #215 officially closing the issue and marking it's completion, I also got my team together to discuss how we will approach, the lecture topic tasks for this milestone.

**Comment by Jjulianirizarry24** [comment]
Completed a task for the first sprint. Ensured team had tasks completed for first sprint. Created lecture topic tasks. Discussed with my team further plans for the current milestone.

**Comment by ghost** [comment]
I haven't been able to work extensively on the project this week, but I have time this weekend, so I will take advantage of that.

**Comment by RaquelQuinones** [comment]
I have not been able to work a lot on the project, due to exams; but I will work on the project for next week.

**Comment by JuanJRivera** [comment]
Work on the implementation of an appendix to the documentation. As indicated in the feedback given by the teacher.

**Comment by joel-alvarado** [comment]
Designed a workflow diagram for the e-mail verification system to better define how it will work.

**Comment by lex939** [comment]
Uploaded more documentation issues and Lecture Topic tasks. Will continue working, with more issues to upload soon.

**Comment by misaelmercado1** [comment]
Created button component for list of locations in figma .

**Comment by jonVargas1** [comment]
I discussed posible lecture topic tasks with my teammates and opened an issue.

**Comment by Chosen-Juan1** [comment]
I revised the documentation and finished a task regarding it.

**Comment by NathBat09** [comment]
Discussed with my team about the lecture topic task, and researched how to complete a task.

**Comment by johnsantiago9** [comment]
Worked in a documentation tasked while I wait to have back my computer to continue my other tasks

**Comment by JorgeRivera94** [comment]
This week, I attended our team meeting where lecture topic tasks were discussed. As for my development progress, I have been inactive due to multiple projects and deliverables. I have completed one issue this week, but it still needs posting.

**Comment by williammartinez10** [comment]
Finished work on #220  and began looking into #237  and #246.

**Comment by yadrielsamir** [comment]
Conduct weekly meeting for admin team and work on admin calendar implementation.
